CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [ENHANCEMENT] Added new ring related config `-distributor.excluded-zones` when set this will exclude the comma-separated zones from the ring, default is "". #4539
 * [ENHANCEMENT] Upgraded Docker base images to `alpine:3.14`. #4514
 * [ENHANCEMENT] Updated Prometheus to latest. Includes changes from prometheus#9239, adding 15 new functions. Multiple TSDB bugfixes prometheus#9438 & prometheus#9381. #4524
+* [BUGFIX] Updated documentation with more details about `-frontend.downstream_url`. #3039
```

Comment:
I don't think we typically up a CHANGELOG item for doc update (@bboreham correct me if I'm wrong). But if we do, I would say this change is more of an `ENHANCEMENT` than `BUGFIX` :-)